avaframe/ana3AIMEC/aimecTools.py

```diff
@@ -1260,6 +1266,28 @@ def scalePathWithCellSize(rasterTransfo, cellSizeSL):
     return rasterTransfo
 
 
+def addSurfaceParalleCoord(rasterTransfo):
+    """ Add the surface parallel coordinate
```

Comment:
along flow path that takes elevation into account - just to add a bit more info?